src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/pipeline_management.yml

```
@@ -0,0 +1,886 @@
+                  - cloudformation:ExecuteChangeSet
+                  - cloudformation:SetStackPolicy
+                  - cloudformation:ValidateTemplate
+                Resource:
+                  - "*"
+              - Effect: Allow
+                Sid: "PassRole"
+                Action:
+                  - 'iam:PassRole'
+                Resource:
+                  - !GetAtt ADFPipelineMangementCloudFormationRole.Arn
+                Condition:
+                  StringEqualsIfExists:
+                    'iam:PassedToService':
+                      - cloudformation.amazonaws.com
+
+  ADFPipelineMangementCloudFormationRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - cloudformation.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Path: /
+      Policies:
+        - PolicyName: "adf-codepipeline-creation"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "codepipeline:CreatePipeline"
+                  - "codepipeline:DeletePipeline"
+                  - "codepipeline:DeleteWebhook"
+                  - "codepipeline:DeregisterWebhookWithThirdParty"
+                  - "codepipeline:GetPipeline"
+                  - "codepipeline:GetPipelineState"
+                  - "codepipeline:PutWebhook"
+                  - "codepipeline:RegisterWebhookWithThirdParty"
+                  - "codepipeline:StartPipelineExecution"
+                  - "codepipeline:TagResource"
+                  - "codepipeline:UpdatePipeline"
+                Resource:
+                  - !Sub arn:${AWS::Partition}:codepipeline:${AWS::Region}:${AWS::AccountId}:webhook:adf-webhook-*
+                  - !Sub arn:${AWS::Partition}:codepipeline:${AWS::Region}:${AWS::AccountId}:${PipelinePrefix}*
+              - Effect: Allow
+                Action:
+                  - "codebuild:CreateProject"
+                  - "codebuild:DeleteProject"
+                  - "codebuild:UpdateProject"
+                Resource:
+                  - !Sub arn:${AWS::Partition}:codebuild:${AWS::Region}:${AWS::AccountId}:project/adf-build-*
+              - Effect: Allow
+                Action:
+                  - "sns:DeleteTopic"
+                  - "sns:CreateTopic"
+                  - "sns:Unsubscribe"
+                  - "sns:Subscribe"
+                  - "sns:SetTopicAttributes"
+                  - "sns:GetTopicAttributes"
+                  - "sns:TagResource"
+                Resource:
+                  - !Sub arn:${AWS::Partition}:sns:${AWS::Region}:${AWS::AccountId}:${PipelinePrefix}*
+              - Effect: Allow
+                Action:
+                  - "iam:CreateRole"
+                  - "iam:CreateRolePolicy"
```

This action does not exist, do you mean AttachRolePolicy?

```suggestion
                  - "iam:AttachRolePolicy"
```